snip-gui/src/app/Components/snippet-panel/snippet-panel.component.html

```diff
@@ -20,11 +20,13 @@
     <!-- editor-->
       <ngx-codemirror
       [(ngModel)]="snippet.content"
-  [options]="codeStyleOptions"
-  placeholder="Hello world!"
+      [options]="codeStyleOptions"
+      placeholder="Hello world!"
       ></ngx-codemirror>
     </div>
   </div>
+  <button mat-button (click)="highlight()">Highlight</button>
```

Comment:
Just added this button for testing purposes, we can delete it once the comments are connected